### DISCUSSION TITLE
Shuffle all tracks

### DISCUSSION THREAD

**Discussion by mischiamnesiac** [namidaco/namida#208] [Q&A]
I've looked for a while but couldn't find a solution. Is there a way to shuffle all tracks in my library? Under extras I chose Shuffle as the floating action button. When I click on shuffle in the track overview (where it lists all tracks in my library), there are only 1 000 tracks added to the playlist. Clicking it again doesn't work, it only creates a different random playlist with 1 000 entries. Is this a fixed setting in the source code, or can it be changed in the app?

**Comment by MSOB7YY** [comment]
queues by default are limited to 1000 for performance concerns, one way to bypass this is
1. play 1 track
2. select all tracks u want to add
3. open dialog and add to queue (play next/last)
4. open miniplayer queue and press shuffle (long press to configure)
